notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>* Remove "fmt: on" and "fmt: off"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>If we remove this, <code>@tf.function</code> is formatted like this, and I think it is a bit redundant. </p><p>What do you think?  </p><p> </p><pre class="ql-syntax" spellcheck="false">@tf.function(
&nbsp; &nbsp; input_signature=[
&nbsp; &nbsp; &nbsp; &nbsp; tf.TensorSpec(
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; None,
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; dtype=tf.string,
&nbsp; &nbsp; &nbsp; &nbsp; )
&nbsp; &nbsp; ]
)
def predict_from_filename(filenames):
</pre><p> </p>

Reply:
@BenoitDherin Please let me know what you think :)

Reply:
<p>Oh! I see. Yes. please disregard my comment then. </p>